scripts/multicluster/rbac/role.yaml

```diff
@@ -0,0 +1,50 @@
+
+---
```

Comment:
If there's no ClusterRoleBinding, I don't think this does anything

Reply:
These manifests are referred from https://github.com/kubernetes-sigs/mcs-api/tree/master/config. ClusterRole is not used in this demo. We can keep it for future use.